.github/workflows/build.yml

```diff
@@ -78,8 +78,18 @@ jobs:
       run: |
         docker run -di --name emscripten -v $(pwd):/src emscripten/emsdk:latest bash
         docker exec emscripten emcc -v
-        docker exec emscripten emcmake cmake .
-        docker exec emscripten make -j 2 VERBOSE=1
+        docker exec emscripten emcmake cmake -B emscripten -DWERROR=ON -DBUILD_TESTS=OFF
+        docker exec emscripten bash -c "cd /src/emscripten && emmake make -j $(nproc)"
```

Comment:
Why is this `cd` needed now but not before?

You don't need to run emmake there..  all this emscripten stuff should already be handled by `emcmake` above and embedded into the resulting `Makefile`.

Reply:
Because before cmake was out at project root, I moved the cmake out to ./emscripten folder, which is planned for store build file for emcc(according to gitignore file).